_posts/2015-11-05-aerogear-unified-push-110-is-out.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: news
+section: news
+title: AeroGear UnifiedPush Server 1.1.0 released!
+author: matzew
+module: push
+releases: 1.1.0.Final
+---
+
+The AeroGear team is happy to announce the version 1.1.0 of the UnifiedPush Server! Over the last year we have developed several new features on our 1.1.0 release series, with a two alpha and four beta releases. The highlight of the release The release focuses on a few new features
```

Comment:
`The highlight of the release The release focuses on a few new features` There is something wrong here :)